packages/app-next/src/App.tsx

```diff
@@ -91,8 +120,14 @@ const app = createApp({
     techdocsPlugin,
     userSettingsPlugin,
     homePlugin,
+    ...collectedLegacyPlugin,
```

Comment:
(just so you can simply fix it)

```suggestion
    ...collectedLegacyPlugins,
```